templates/minimal/node/README.md

```diff
@@ -0,0 +1,38 @@
+# Node
+
+A blockchain node is an application that allows users to participate in a
```

Comment:
https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/wasm_meta_protocol/index.html is a good resource here.

Reply:
That was helpful.